master/buildbot/www/hooks/bitbucketcloud.py

```diff
@@ -70,7 +70,10 @@ def _get_payload(self, request):
 
     def handle_repo_push(self, payload):
         changes = []
+        if payload['repository'].get('project'):
```

Comment:
there is a syntax error here.

Please simplify:

```suggestion
        project = payload['repository'].get('project', {'name': 'none'})['name']:
```